sys/stats/full/include/stats/stats.h

```diff
@@ -48,6 +48,7 @@ struct stats_hdr {
     int s_map_cnt;
 #endif
     STAILQ_ENTRY(stats_hdr) s_next;
+    uint8_t data[0];
```

Comment:
The C standard prohibits zero-length arrays (gcc allows them as an extension).  The pedantically correct way to do this is to use a "flexible array member", i.e.,
```
uint8_t data[]
```